execute_pipeline.py

```diff
@@ -82,6 +109,18 @@ def _CreateArgumentParser():
       type=str,
       default="..",
       help="Root directory where the input, output, and tool repositories live")
+  parser.add_argument(
+      "--local_repo",
+      type=str,
+      default=None,
+      help="Directory where local proto and gapic configs lives. This flag is "
+           "used for remote execution only.")
+  parser.add_argument(
+        "--env",
+        type=str,
+        default="LOCAL",
+        help="Specify which environment should be used to execute pipeline. "
+             "Valid values are 'LOCAL' and 'REMOTE'")
```

Comment:
I'm not sure why you switched from a boolean flag (--remote-mode) to a string flag -- but doing this, you should add choices (see https://docs.python.org/2/library/argparse.html#choices)